.github/release-drafter.yml

```diff
@@ -23,8 +27,13 @@ categories:
   - title: 'Installation'
     labels:
       - 'docker'
-      - 'installation'
+      - 'install'
       - 'dependencies'
+  - title: 'Miscellaneous'
```

Comment:
Are those properties of commits?
I would say that these only makes sense during a PR, and should not be preserved in the logs?

Reply:
Not sure I understand this question? the labels are the github labels (?)